### DISCUSSION TITLE
v1.37 really requires at least Home Assistant 2024.4?

### DISCUSSION THREAD

**Discussion by bartplessers** [Koenkk/zigbee2mqtt#22577] [General]
On the release notes, I read
_For Home Assistant users: this release requires at least Home Assistant 2024.4._

However, I can't upgrade right now because I'm stuck for another reason: https://github.com/home-assistant/core/issues/115056

So does v1.37 really requires at least Home Assistant 2024.4?
What will happen if I upgrade to v1.37 if I'm on Home Assistant 2024.3.3?

**Comment by Koenkk** [comment]
> What will happen if I upgrade to v1.37 if I'm on Home Assistant 2024.3.3?

The only thing that might break is the lights (they become unavailable, at least that is what happened with 2024.1)
